gbpservice/nfp/service_plugins/firewall/nfp_fwaas_plugin.py

```diff
@@ -98,6 +98,7 @@ def _is_net_reachable_from_net(self, context, tenant_id, from_net_id,
     @param to_net_id: the destination network for the search
     @return: True or False whether a path exists
     """
+    original_context = context
```

Comment:
Currently how we are maintaining, as the file contains this extraroute patch ?
We can follow the approach which we used in orchestration driver to identify setup mode whether it is neutron or aci.